arch/arm/dts/zynq-sidekiqz2.dts

```diff
@@ -0,0 +1,77 @@
+/*
+ * Avnet PicoZED SDR2 board DTS
+ *
```

Comment:
This is not the Avnet PicoZED SDR2 board...

Reply:
good catch ; will fix :)